src/test/java/data/CameraShotTest.java

```diff
@@ -44,5 +44,10 @@ public void setDescriptionTest() {
         cameraShot.setDescription("a");
         assertEquals("a", cameraShot.getDescription());
     }
+    
+    @Test
+    public void toStringTest() {
+        assertEquals("CameraShot()", cameraShot.toString());
+    }
 
 }
```

Comment:
Github tells me there's something wrong here. Not sure if there is, but thought I'd mention it.